AppController/djinn.rb

```diff
@@ -486,7 +490,8 @@ class Djinn
     'write_detailed_processes_stats_log' => [TrueClass, 'False', true],
     'write_detailed_proxies_stats_log' => [TrueClass, 'False', true],
     'zone' => [String, nil, true],
-    'fdb_clusterfile_content' => [String, nil, true]
+    'fdb_clusterfile_content' => [String, nil, true],
+    'update' => [String, nil, false]
```

Comment:
Assuming you don't serialize twice, the class should be `Array` here, and it would probably make sense for the default value to be `[]`.